DNN Platform/Tests/DotNetNuke.Tests.Data/DotNetNuke.Tests.Data.csproj

```diff
@@ -74,7 +74,7 @@
       <Private>True</Private>
     </Reference>
     <Reference Include="PetaPoco">
-      <HintPath>..\..\Components\PetaPoco\bin\PetaPoco.dll</HintPath>
+      <HintPath>..\..\Library\bin\PetaPoco.dll</HintPath>
```

Comment:
Instead of referencing the `Library\bin\PetaPoco.dll` could we update the test project to use the nuget file. This will limit a dependency on the compiled dll and force NuGet to resolve the dependency which I believe is the best practice we should be following

Reply:
I had the same thought, however the reasoning behind my initial decision was to try and keep the dependabot pull requests down to a minimum. Since they only look at a single folder/configuration file and don't do a recursive search. We'd end up with 3 pull requests to update the reference in the Dnn.Platform project. 

I'm happy to make this change, if that's something everyone agrees on. However, I think we'd be OK with the single nuget reference and then just using a standard file based reference on the support projects.

Another thought (untested and not researched), we might be able to do something similar with the yarn-workspace monorepo configuration so we can decrease the PR workload when these references are updated.

Reply:
I'm going to go ahead and merge, but I'd like to see us using NuGet for references where possible.  We'll need to investigate if there's a better way to manage that with Dependabot